packages/plugin-sales-api/src/exporter.ts

```
@@ -470,9 +470,73 @@ const fillValue = async (
+          data: { _id: id },
+          isRPC: true,
+          defaultValue: {}
+        });
+
+        customerRows.push(customer);
+      }
+
+      value = customerRows
+        .map(customer => {
+             return customer.primaryPhone || 
+             (customer.phones && customer.phones.length ? customer.phones[0] : '');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)